cmd/convert.go

```diff
@@ -138,10 +130,14 @@ func newConvertCmd() *cobra.Command {
 
 			if stat, err := os.Stat(outputPath); !errors.Is(err, os.ErrNotExist) && stat.IsDir() {
 				// output path is a directory so write a new file to it
-				srcFileName := filepath.Base(srcPath)
-				srcFileName = strings.TrimSuffix(srcFileName, filepath.Ext(srcFileName))
+				outFileName := filepath.Base(srcPath)
+				if srcPath == os.Stdin.Name() {
+					outFileName = defaultOutputFileName
+				}
 
-				outputPath = filepath.Join(outputPath, srcFileName)
+				outFileName = strings.TrimSuffix(outFileName, filepath.Ext(outFileName))
+
+				outputPath = filepath.Join(outputPath, outFileName)
```

Comment:
II am guessing not since the outputPath would then change

Reply:
see above